docs/api/non_geo_datasets.csv

```diff
@@ -37,6 +37,8 @@ Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
 `SpaceNet`_,I,WorldView-2/3 Planet Lab Dove,"1,889--28,728",2,102--900,0.5--4,MSI
 `SSL4EO`_-L,T,Landsat,1M,-,264x264,30,MSI
 `SSL4EO`_-S12,T,Sentinel-1/2,1M,-,264x264,10,"SAR, MSI"
+`SSL4EO-L Benchmark`_, S, Lansat & CDL, 25k,18,264x264,30,MSI
+`SSL4EO-L Benchmark`_, S, Lansat & NLCD, 25k,14,264x264,30,MSI
```

Comment:
NLCD has 17 classes and CDL has 134 classes (although we only evaluated on a subset in our paper). Also, I would remove the spaces around commas and capitalize the K.

Reply:
the config files that we used say 14 classes for nlcd and 18 for cdl